<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-01-31T18:49:03 IST" name="Default suite" finished-at="2025-01-31T18:49:36 IST" duration-ms="33108">
    <groups>
    </groups>
    <test started-at="2025-01-31T18:49:03 IST" name="Default test" finished-at="2025-01-31T18:49:36 IST" duration-ms="33108">
      <class name="com.wise.tests.EndtoEndCases">
        <test-method is-config="true" signature="lunchBrowser()[pri:0, instance:com.wise.tests.EndtoEndCases@243c4f91]" started-at="2025-01-31T18:49:03 IST" name="lunchBrowser" finished-at="2025-01-31T18:49:06 IST" duration-ms="3691" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lunchBrowser -->
        <test-method signature="performLogin()[pri:0, instance:com.wise.tests.EndtoEndCases@243c4f91]" started-at="2025-01-31T18:49:06 IST" name="performLogin" description="Perform login as a tutor" finished-at="2025-01-31T18:49:11 IST" duration-ms="4358" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- performLogin -->
        <test-method signature="classRoom()[pri:0, instance:com.wise.tests.EndtoEndCases@243c4f91]" started-at="2025-01-31T18:49:11 IST" name="classRoom" description="Go to the classroom" finished-at="2025-01-31T18:49:12 IST" depends-on-methods="com.wise.tests.EndtoEndCases.performLogin" duration-ms="1227" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- classRoom -->
        <test-method signature="scheduleSession()[pri:0, instance:com.wise.tests.EndtoEndCases@243c4f91]" started-at="2025-01-31T18:49:12 IST" name="scheduleSession" description="Schedule a session" finished-at="2025-01-31T18:49:35 IST" depends-on-methods="com.wise.tests.EndtoEndCases.classRoom" duration-ms="22927" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scheduleSession -->
        <test-method signature="assertSession()[pri:0, instance:com.wise.tests.EndtoEndCases@243c4f91]" started-at="2025-01-31T18:49:35 IST" name="assertSession" description="Assert the session" finished-at="2025-01-31T18:49:36 IST" depends-on-methods="com.wise.tests.EndtoEndCases.scheduleSession" duration-ms="859" status="FAIL">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[host Name expected [[[ChromeDriver: chrome on linux (88c9d3374f35469953718eb508df9f19)] -> css selector: .session .text-uppercase]] but found [HOSTED BY WISE TESTER]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: host Name expected [[[ChromeDriver: chrome on linux (88c9d3374f35469953718eb508df9f19)] -> css selector: .session .text-uppercase]] but found [HOSTED BY WISE TESTER]
at com.wise.pagefactory.GroupCoursesPage.verifyTheHostName(GroupCoursesPage.java:158)
at com.wise.tests.EndtoEndCases.assertSession(EndtoEndCases.java:44)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- assertSession -->
      </class> <!-- com.wise.tests.EndtoEndCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
